src/components/DomEvents.js

```diff
@@ -137,8 +163,14 @@ function DomEvents() {
     if (node) {
       previewRef.current = node;
       const eventListeners = addLoggingEvents(node, (event) => {
-        event.id = buffer.current.length;
-        buffer.current.push(event);
+        const log = {
+          id: buffer.current.length + 1,
+          type: event.event.EventType,
+          name: event.event.name,
+          element: event.target.tagName,
+          selector: event.target.toString(),
+        };
+        buffer.current.push(log);
```

Comment:
As the log is still being pushed to the end of the buffer, I'm curious. How does this work?

Do we sort on every call?

Reply:
For now we sort because in my initial idea I wanted to sort for everything but as you said it dosn't make sense.
I think that in this case I can leave the hook but don't use it. Change this line of code pushing at the beginning or at the end depending on order